console/.gitignore

```diff
@@ -1,4 +1,4 @@
 #node_modules
 node_modules
-
+package-lock.json
```

Comment:
Why ignored? According to [npm documentation](https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md), this file is intended to be commited for several reasons.

[For more](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5)